### PR TITLE
Adding tox env for pubsub sys tests w / emulator.

### PR DESCRIPTION
**Pull request by dhermes**
Modeled after #1393.

Also fixes a small docs bug in `CONTRIBUTING`.

### CODE REVIEW

File: scripts/pubsub_emulator.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    # Ignore stdin and stdout, don't pollute the user's output with them.
+    proc_start = subprocess.Popen(_START_CMD, stdout=subprocess.PIPE,
+                                  stderr=subprocess.PIPE)
+    try:
+        env_lines = subprocess.check_output(
+            _ENV_INIT_CMD).strip().split('\n')
+        host, = [line.split(_HOST_LINE_PREFIX, 1)[1] for line in env_lines
+                 if line.startswith(_HOST_LINE_PREFIX)]
+        # Set environment variables before running the system tests.
+        os.environ[PUBSUB_EMULATOR] = host
```
Comment (tseaver) [inline scripts/pubsub_emulator.py]: Should any of this be factored out to share w/ the datastore emulator script?

Comment (dhermes) [inline scripts/pubsub_emulator.py]: Probably. I thought about doing it but then soldiered on here.

Will rebase on top of #1401 and then refactor.

**Comment by dhermes** [comment]
@tseaver PTAL. I re-factored the `datastore_emulator` script so it could be used for `pubsub` as well.

**Comment by tseaver** [comment]
LGTM

**Pull request merged**